test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -874,31 +875,16 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox app %s', () => {
     expect(stackFrames).toEqual(
       process.env.TURBOPACK
         ? [
-            // TODO: Why is Turbopack off by one in the column?
```

Comment:
Why remove this TODO? There's still a one column difference between Turbopack and Webpack.

Reply:
I thought we know the reason already, it's 0 and 1 based index and we have the different everywhere in other errors tests. We can change the TODO to sth unify the index or merge snapshot

Reply:
We haven't confirmed that and even if, one of them is still a bug. The code location should be consistent across bundlers, environments etc.

Reply:
I kept the TODO but just updated the message